com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeVolumeAsset.cs

```diff
@@ -145,6 +145,11 @@ static int AlignUp16(int count)
             return true;
         }
 
+        internal bool ResolveCells(TextAsset cellDataAsset, TextAsset cellOptionalDataAsset, TextAsset cellSharedDataAsset, TextAsset cellSupportDataAsset)
```

Comment:
Note, this is a leftover from old version of the change too, but somewhat low impact and useful for prototyping ideas.